apps/web/vibes/2px/components/dropdown/index.tsx

```diff
@@ -0,0 +1,94 @@
+'use client'
+
+import { useState } from 'react'
+
+import clsx from 'clsx'
+
+import { ChevronDownIcon } from '../icons'
+
+interface OptionProps {
+  label: string
+  value: string
+}
+
+interface DropdownProps {
+  placeholder: string
+  options: OptionProps[]
+  status?: 'normal' | 'error' | 'success'
+  disabled?: boolean
+  value?: string
+  setValue: (value: string) => void
+  className?: string
+}
+
+const Dropdown = ({
+  placeholder,
+  options,
+  status = 'normal',
+  disabled,
+  setValue,
+  value,
+  className,
+}: DropdownProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className={clsx('relative w-full font-body text-foreground @container', className)}>
+      <button
+        className={clsx(
+          'relative z-0 flex w-full items-center justify-between border-2 px-4 py-[1.125rem] text-base font-medium leading-6 outline-none @lg:px-5 @lg:py-[1.375rem] @lg:text-xl @lg:leading-9',
+          {
+            'border-error': status === 'error',
+            'border-success': status === 'success',
+            'border-foreground': status === 'normal',
+            'cursor-not-allowed border-contrast-300 text-contrast-300': disabled,
+            'hover:border-dashed focus:border-dashed': !disabled,
+            'border-dashed': isOpen,
+          }
+        )}
+        disabled={disabled}
+        onClick={() => setIsOpen(!isOpen)}
+        type="button"
+        aria-label={placeholder}
+      >
+        {options.find(option => option.value === value)?.label || placeholder}
+
+        <ChevronDownIcon className="stroke-foreground" />
+      </button>
+
+      <div
+        className={clsx(
+          'fixed left-0 top-full z-10 mt-2 flex w-full flex-col items-start gap-1 border-2 border-foreground bg-background px-8 py-6 text-base font-medium leading-6 @lg:text-xl @lg:leading-9',
```

Comment:
I believe the the fonts should be of `sm` and `2xl` sizes for small and big screens, respectively.

Reply:
Also, the larger screens should have a `letter-spacing: -0.0175rem;` according to the design.

Reply:
Done 🙏